pkg/server/options/authentication.go

```diff
@@ -198,11 +198,17 @@ func createKubeConfig(kcpAdminToken, shardAdminToken, userToken, baseHost, tlsSe
 			CertificateAuthorityData: caData,
 			TLSServerName:            tlsServerName,
 		},
+		"system:admin": {
+			Server:                   baseHost + "/clusters/system:admin",
+			CertificateAuthorityData: caData,
+			TLSServerName:            tlsServerName,
+		},
 	}
 	kubeConfig.Contexts = map[string]*clientcmdapi.Context{
 		"root":         {Cluster: "root", AuthInfo: kcpAdminUserName},
 		"base":         {Cluster: "base", AuthInfo: kcpAdminUserName},
-		"system:admin": {Cluster: "base", AuthInfo: shardAdminUserName},
+		"system:admin": {Cluster: "system:admin", AuthInfo: shardAdminUserName},
+		"system:base":  {Cluster: "base", AuthInfo: shardAdminUserName},
```

Comment:
the colon notation is confusing IMO. shard-base or shard-admin-base?

Reply:
I like `shard-base`. Will adjust.